ephios/core/forms/users.py

```diff
@@ -118,11 +119,11 @@ class GroupForm(PermissionFormMixin, ModelForm):
         required=False,
     )
     is_management_group = PermissionField(
```

Comment:
I think, as we are disabling/force-checking the other checkboxes when this is checked, we should also hide the group selection for "allow publish for" when `is_management_group` gets checked.

Reply:
I opted to hide it instead of disabling it to avoid confusion as there aren't all groups selected (which is not what actually happens permission-wise) but the field wouldn't accept changes